client/blocks/site/index.jsx

```diff
@@ -151,9 +151,14 @@ class Site extends React.Component {
 								: site.domain }
 						</div>
 						{ /* eslint-disable wpcalypso/jsx-gridicon-size */ }
-						{ this.props.site.is_private && (
-							<span className="site__badge site__badge-private">{ translate( 'Private' ) }</span>
-						) }
+						{ this.props.site.is_private &&
+							( this.props.site.is_coming_soon ? (
+								<span className="site__badge site__badge-private">
+									{ translate( 'Coming soon' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **62** times:
`translate( 'Coming Soon' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feature/coming-soon-support)